src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/UnusedImportsCheckTest.java

```diff
@@ -108,21 +108,21 @@ public void testProcessJavadoc() throws Exception {
     public void testProcessJavadocWithLinkTag() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(UnusedImportsCheck.class);
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getPath("InputUnusedImportWithValueTag.java"), expected);
+        verify(checkConfig, getPath("InputUnusedImportsWithValueTag.java"), expected);
     }
 
     @Test
     public void testAnnotations() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(UnusedImportsCheck.class);
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getNonCompilablePath("package-info.java"), expected);
+        verify(checkConfig, getNonCompilablePath("InputUnusedImportsAnnotations.java"), expected);
```

Comment:
@romani It is still an input file. If test doesn't rely on requiring a specific file name, why must we keep it like that?